### PR TITLE
Add alert for RDS BurstBalance per skyscrapers/engineering#1009

### PR DESCRIPTION
**Pull request by simonrondelez**
# Description

# Related Issue

# Motivation and Context

# How Has This Been Tested?

# Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

# Checklist

- [ ] I have followed the PR process as described [here](https://github.com/skyscrapers/documentation/blob/master/coding_guidelines/git.md#pull-requests)
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: rds-monitoring/templates/prometheusrules.yaml [hunk 1]
```diff
@@ -94,3 +94,12 @@ spec:
           annotations:
             message: 'Replica lag for RDS instance {{`{{ $labels.dbinstance_identifier }}`}} is higher than 30 seconds.'
             runbook_url: 'https://github.com/skyscrapers/documentation/tree/master/runbook.md#alert-name-rdsreplicalaghigh'
+        - alert: RDSBurstBalanceLow
+          expr: aws_rds_burstbalance_average{job="{{ .Release.Name }}-prometheus-cloudwatch-exporter"} {{ include "rds-monitoring.filterOnTag" . }} < 20
+          for: 10m
+          labels:
+{{ include "rds-monitoring.alertLabels" . | indent 12 }}
+            severity: info
```
Comment (iuriaranda) [inline rds-monitoring/templates/prometheusrules.yaml]: Hmmm shouldn't this be higher severity? Seems quite important to me no?

Comment (simonrondelez) [reply rds-monitoring/templates/prometheusrules.yaml]: what do you propose? Warning? I wouldn't do critical for this.

Comment (iuriaranda) [reply rds-monitoring/templates/prometheusrules.yaml]: idd, probably not critical... :thinking: 
then let's go for warning

**Review by iuriaranda — APPROVED** [review]

File: rds-monitoring/templates/prometheusrules.yaml (hunk 1 above)
Comment (simonrondelez) [reply rds-monitoring/templates/prometheusrules.yaml]: done

**Pull request merged**